src/serviceconnector-passwordless/azext_serviceconnector_passwordless/_params.py

```diff
@@ -68,6 +73,7 @@ def load_arguments(self, _):
             add_local_connection_block(c)
             add_customized_keys_argument(c)
             c.argument('yes', arg_type=yes_arg_type)
+            c.argument('new', arg_type=new_arg_type)
```

Comment:
would it make sense to add a parameter like --force-cleanup etc in delete command instead of add --new in create command? what's your thoughts?